vue/sbc-common-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sbc-common-components",
-  "version": "2.4.16",
+  "version": "2.4.17",
```

Comment:
You need to run `npm i` after changing this, so that the package-lock.json file gets updated.

Reply:
And please commit the package lock file. This ensures that the actual packages used by build process are identical to the packages you used locally.